flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -696,7 +697,24 @@ private void stopTaskExecutorServices() throws Exception {
                             partitionStateChecker,
                             getRpcService().getExecutor());
 
-            taskMetricGroup.gauge(MetricNames.IS_BACKPRESSURED, task::isBackPressured);
+            taskMetricGroup.gauge(MetricNames.IS_BACK_PRESSURED, task::isBackPressured);
```

Comment:
`isBackPressured` is used more often now which might lead to many `volatile` reads (`Future.isDone`).
However, these reads can be eliminated by exiting the loop as soon as some future is not done (instead of `!allOf(futures).isDone`).
It would also make the code more readable IMO.
WDYT?